GCC 13 memcpy warning in can_message.hpp

When compiling modm for an STM32F427 using arm-none-eabi-gcc 13.2.rel1-2, I get the following warnings:

```
src/modm/platform/can/can_1.cpp: In static member function 'static bool modm::platform::Can1::getMessage(modm::can::Message&, uint8_t*)':
src/modm/platform/can/can_1.cpp:301:23: warning: 'void* memcpy(void*, const void*, size_t)' writing to an object of type 'struct modm::can::Message' with no trivial copy-assignment; use copy-assignment or copy-initialization instead [-Wclass-memaccess]
  301 |                 memcpy(&message, &rxMessage.message, sizeof(message));
      |                 ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from src/modm/architecture/interface/can.hpp:18,
                 from src/modm/platform/can/can_1.hpp:19,
                 from src/modm/platform/can/can_1.cpp:25:
src/modm/architecture/interface/can_message.hpp:27:8: note: 'struct modm::can::Message' declared here
   27 | struct Message
      |        ^~~~~~~
src/modm/platform/can/can_2.cpp: In static member function 'static bool modm::platform::Can2::getMessage(modm::can::Message&, uint8_t*)':
src/modm/platform/can/can_2.cpp:303:23: warning: 'void* memcpy(void*, const void*, size_t)' writing to an object of type 'struct modm::can::Message' with no trivial copy-assignment; use copy-assignment or copy-initialization instead [-Wclass-memaccess]
  303 |                 memcpy(&message, &rxMessage.message, sizeof(message));
      |                 ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from src/modm/architecture/interface/can.hpp:18,
                 from src/modm/platform/can/can_2.hpp:19,
                 from src/modm/platform/can/can_2.cpp:25:
src/modm/architecture/interface/can_message.hpp:27:8: note: 'struct modm::can::Message' declared here
   27 | struct Message
      |        ^~~~~~~
```

--- a/src/modm/platform/can/lpc/c_can.cpp.in
+++ b/src/modm/platform/can/lpc/c_can.cpp.in
@@ -311,7 +311,7 @@ modm::platform::Can::getMessage(can::Message & message)
 		return false;
 	}
 	else {
-		memcpy(&message, &rxQueue.get(), sizeof(message));
+		message = rxQueue.get();
 		rxQueue.pop();
 
 		// Check for other messages in MOBs

--- a/src/modm/platform/can/stm32/can.cpp.in
+++ b/src/modm/platform/can/stm32/can.cpp.in
@@ -391,7 +391,8 @@ modm::platform::Can{{ id }}::getMessage(can::Message& message, uint8_t *filter_i
 	}
 	else {
         auto& rxMessage = rxQueue.get();
-		memcpy(&message, &rxMessage.message, sizeof(message));
+		message = rxMessage.message;
+
         if(filter_id != nullptr) (*filter_id) = rxMessage.filter_id;
 		rxQueue.pop();
 		return true;

